Utils/Dataflow/060_upload2virtuoso/uploadTTL.sh

```diff
@@ -114,16 +123,19 @@ upload_files () {
         ;;
     esac
 
-    eval "$cmd" || { echo "(ERROR) An error occured while uploading file: $INPUTFILE" >&2; continue; }
-
+    eval "$cmd" || { echo "(ERROR) An error occured while uploading file: $INPUTFILE" >&2; continue; } &
+    echo -ne EOMessage
+    echo -ne EOProcess
```

Comment:
General comment: dollar sign is missed before variable names here and there (`echo EOProcess` will output `"EOProcess"` string, not the value of the `EOProcess` variable).

Reply:
@mgolosova 
Thank you. I apologize for that. Will fix it!